templates/dialog-pocket-change.html

```diff
@@ -0,0 +1,8 @@
+<div class='pocket-change flexrow' 
```

Comment:
Doesn't look like this file is being used anywhere

Reply:
removed